x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -185,7 +185,7 @@ export interface RuleEcs {
   id?: string[];
   rule_id?: string[];
   name?: string[];
-  false_positives: string[];
+  false_positives?: string[];
```

Comment:
This change is necessary to allow integration with security and observability that define this as possible undefined on their own ECS types.